forge/ee/routes/tables/index.js

```diff
@@ -0,0 +1,303 @@
+                }
+            },
+            response: {
+                200: {
+                    type: 'object',
+                    $ref: 'DatabaseCredentials'
+                },
+                '4xx': {
+                    $ref: 'APIError'
+                },
+                500: {
+                    $ref: 'APIError'
+                }
+            }
+        }
+    }, async (request, reply) => {
+        try {
+            const creds = await app.tables.getDatabase(request.team, request.params.databaseId)
+            if (creds) {
+                reply.send(await app.db.views.Table.table(creds))
+            } else {
+                reply.status(404).send({ code: 'not_found', error: 'Database not found' })
+            }
+        } catch (err) {
+            // console.log(err)
+            reply.status(500).send({ code: 'unexpected_error', error: 'Failed to retrieve database' })
+        }
+    })
+
+    /**
+     * @name /api/v1/teams/:teamId/databases/:databaseId
+     * @description Delete a specific team database by ID
+     * @static
+     * @memberof forge.routes.api.team.tables
+     */
+    app.delete('/:databaseId', {
+        preHandler: app.needsPermission('team:database:delete'),
+        schema: {
+            summary: '',
+            tags: ['FF tables'],
+            params: {
+                type: 'object',
+                properties: {
+                    databaseId: { type: 'string' }
+                }
+            },
+            response: {
+                200: {},
+                '4xx': {
+                    $ref: 'APIError'
+                },
+                500: {
+                    $ref: 'APIError'
+                }
+            }
+        }
+    }, async (request, reply) => {
+        try {
+            await app.tables.destroyDatabase(request.team, request.params.databaseId)
+            reply.send({})
+        } catch (err) {
+            // console.log(err)
+            reply.status(500).send({ code: 'unexpected_error', error: 'Failed to destroy database' })
+        }
+    })
+
+    /**
+     * @name /api/v1/teams/:teamId/databases/:databaseId/tables
+     * @description List all tables in a specific team database
+     * @static
+     * @memberof forge.routes.api.team.tables
+     */
+    app.get('/:databaseId/tables', {
+        preHandler: app.needsPermission('team:database:list'),
+        schema: {
+            summary: '',
+            tags: ['FF tables'],
+            params: {
+                type: 'object',
+                properties: {
+                    databaseId: { type: 'string' }
+                }
+            },
+            response: {
+                200: {
+                    type: 'array',
+                    items: {
+                        type: 'object',
+                        properties: {
+                            name: { type: 'string' },
+                            schema: { type: 'string' }
+                        }
+                    }
```

Comment:
I dont think this should return an array as it may turn out we need meta/pagination when the start getting too large.
Changing this to an object now avoids any future duck type checking awkwardness